### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by StevenWafeek**
In this project :
- Create basic UI.
- Create app.rb file It have methods that do the following:
1. List all books.
2. List all people.
3. Create a person (teacher or student, not a plain Person).
4. Create a book.
5. Create a rental.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+def create_person(people)
+  puts "\nCreate a person:"
+  puts "Enter the person's name:"
+  name = gets.chomp
+
+  puts "Enter the person's age:"
+  age = gets.chomp.to_i
+
+  puts 'Is it a teacher or a student? (T/S)'
+  role = gets.chomp.downcase
```
Comment (christianbiring1) [inline app.rb:67]: - [ ] As shown in the demo, this feature is supposed to come before entering the name of the person, which means before typing the name of the person one should know if they are a student or teacher, Kindly make sure you correct for a better User Experience.

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+  person = people.find { |p| p.id == person_id }
+
+  if person.nil?
+    puts "Person with ID #{person_id} not found."
+    return
+  end
+
+  puts "Enter the book's title:"
+  book_title = gets.chomp
```
Comment (christianbiring1) [inline app.rb]: - [ ] For a better user experience, as it is shown in the demo, Kindly make sure while creating a rental, the user can see all the books displayed on the UI, and from there they will be able to choose the one they want to rent Kindly fix that.
You may consider checking again the [demo](https://www.youtube.com/watch?v=vkkgrhD6aXQ) to refresh

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @StevenWafeek , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS" REQUEST CHANGES :recycle: 


## Highlights
- [ ] Great work on implementing the basic UI for the App with all the options available and running :100: 
- [ ] Work documented well :page_facing_up: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Zuhaib042 — APPROVED** [review]
Hi @StevenWafeek 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Well implemented suggested changes
✅ App is working fine
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**